php-checks/src/main/java/org/sonar/php/checks/formatting/IndentationCheck.java

```diff
@@ -76,17 +76,17 @@ private void checkArgumentsIndentation(FormattingStandardCheck formattingCheck,
       int expectedIndentationColumn = getLineStartingColumn(node.getParent()) + PSR2_INDENTATION;
       int callingLine = methodName.getTokenLine();
 
-      if (!isOnSameLine(methodName.getToken(), lastParam)) {
+//      if (!isOnSameLine(methodName.getToken(), lastParam)) {
 
         if (!isCorrectlySplittedOnLines(callingLine, arguments)) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this line start at column 7. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AIndentationCheck)